src/grunt/helper.js

```diff
@@ -231,7 +245,7 @@ ns.init = function (grunt) {
     });
 
     // Single run of unit tests where code coverage is calculated
-    grunt.registerTask('test-coverage', ['karma:coverage', 'post-coverage']);
+    grunt.registerTask('test-coverage', ['forceOn', 'karma:coverage', 'post-coverage', 'forceOff']);
```

Comment:
I'm not sure if I'd go this far. Maybe add another task that is `force-test-coverage`?